### ISSUE TITLE
fabric3 incompatible with versions of paramiko in CF

### ISSUE THREAD

**Issue by bryevdv** [conda-forge/paramiko-feedstock#11]
Hi, I just ran into this issue when trying to make an automated Bokeh release:

http://uucode.com/blog/2015/02/20/workaround-for-ctr-mode-needs-counter-parameter-not-iv/

The relevant comment is:
```
This has been fixed in Paramiko 1.17.4+/1.18.2+/2.0.5+/2.1.2+
```
However, `fabric3` in C-F won't install with `paramiko>=2` and the `0.18.1` is the latest pre-2.0 release in C-F. I confirmed that pip installing `0.18.2` solved the issue. Is it possible to get `0.18.2` into C-F so that fabric3 can function?

**Comment by jakirkham** [comment]
Yes, what version would you like?

**Comment by bryevdv** [comment]
`0.18.2` seems to work

**Comment by jakirkham** [comment]
`0.18.2`? Or `1.18.2`?

**Comment by bryevdv** [comment]
`1.18.2`, sorry

**Comment by jakirkham** [comment]
np

**Comment by jakirkham** [comment]
Added PR ( https://github.com/conda-forge/paramiko-feedstock/pull/12 ) to release 1.18.2.

**Comment by bryevdv** [comment]
Does closing mean that the package should already be available?
```
❯ conda install -c conda-forge paramiko=1.18.2
Fetching package metadata .............

PackageNotFoundError: Package missing in current osx-64 channels:
  - paramiko 1.18.2*
```

**Comment by jakirkham** [comment]
Closing was suppose to happen when the PR was merged, but didn't happen right away. So I closed it a little after. That said, it does appear they are all up there now.

<img width="1552" alt="screen shot 2017-06-30 at 02 45 59" src="https://user-images.githubusercontent.com/3019665/27724124-51411c7c-5d3e-11e7-81ed-ee4efbddafce.png">

Let me know if you are still having issues. Though will be out for a few days because of the holiday.
